controller/DeliveryServiceController.go

```diff
@@ -0,0 +1,41 @@
+package controller
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"log"
+	"net/http"
+	"stlab.itechart-group.com/go/food_delivery/courier_service/dao"
+)
+
+// @Summary  CreateDeliveryService
+// @Tags DeliveryService
+// @Description create a Delivery Service
+// @ID CreateDeliveryService
+// @Accept  json
+// @Produce json
+// @Param input body dao.DeliveryService true "Delivery Service"
+// @Success 200 {object} dao.DeliveryService
+// @Failure 400,404 {string} string
+// @Failure 500 {string} string
```

Comment:
I don't see where your handler can return 404 status

Reply:
And Anna said that we should return object not a string (see here https://github.com/Baraulia/AUTHORIZATION_SERVICE/pull/7)